beetsplug/bpd/__init__.py

```diff
@@ -651,9 +719,10 @@ def send(self, lines):
         if isinstance(lines, six.string_types):
             lines = [lines]
         out = NEWLINE.join(lines) + NEWLINE
-        # Don't log trailing newline:
-        message = out[:-1].replace(u'\n', u'\n' + u' ' * 13)
-        self.server._log.debug('server: {}', message)
+        session = u'>[{}]: '.format(self.address)
+        for l in out.split(NEWLINE)[:-1]:
+            l = l.replace('{', '{{').replace('}', '}}')
+            self.server._log.debug(session + l)
```

Comment:
Here's a way to avoid the need for those `replace`s:

```py
self.server._log.debug('{}', session + l)
```

Reply:
Much nicer!